lib/python/picongpu/plugins/plot_mpl/requirements.txt

```diff
@@ -1,3 +1,6 @@
 matplotlib
 numpy
+scipy
+sys
+getopt
```

Comment:
I am surprised these dependencies were not included beforehand - `scipy` gets included by `-r ../data/requirements.txt` - the others were not included beforehand

Reply:
I was surprised too :D I guess it doesn't hurt, to add them to the `requirements.txt`, right?

Reply:
I agree